packages/utils/provider-amt/build.yaml

```diff
@@ -0,0 +1,18 @@
+image: busybox
+unpack: true
+copy:
+  - image: "{{ .Values.labels.repository }}:v{{ .Values.version }}"
+    source: /usr/local/lib/librpc.so
+    destination: /usr/local/lib/librpc.so
+  - image: "{{ .Values.labels.repository }}:v{{ .Values.version }}"
+    source: /usr/local/include/librpc.h
+    destination: /usr/local/include/librpc.h
+  - image: "{{ .Values.labels.repository }}:v{{ .Values.version }}"
+    source: /system/providers/agent-provider-amt
+    destination: /usr/local/system/providers/agent-provider-amt
+steps:
+  - chmod +x /usr/local/system/providers/agent-provider-amt
```

Comment:
@paynejacob I thought we would use it as a bundle and not as a provider?

Currently the agent doesn't support multiple bundles, and this means you can't use amt with others - or I am missing something?

Reply:
This bundle will install amt as a provider.  The idea being if you are using amt to manage your hardware you would use the os builder to create an iso with the amt bundle installed.  AMT only needs to be configured once so it makes sense to run it on install.  Are you saying we can only have one bundle in the install config or one bundle per system?

Reply:
yes, at the moment we can have only one bundle only - they don't play nicely together yet

Reply:
see https://github.com/kairos-io/kairos/blob/afc1a13ab8053f76f0a0d1c54ebabc4686253abe/internal/agent/install.go#L115 - we should do some merging strategy in all the places were we call providers in order to support multiple providers

Reply:
what about making it a bundle? that should work as well and can also bring binaries with it (thus, you need to place them for instance in a persistent location, as `/usr/local/bin`)

Reply:
I see we want it to be a systemd service, I can make that change.

Reply:
right indeed, no need even to copy anything, if it's a one-shot command, it can just register and be done with it!